COMPILING.md

```diff
@@ -28,14 +30,25 @@ make
 make install
 
 # Compile FieldOpt
-cd ..
+cd ../..
 mkdir cmake-build-debug
 cd cmake-build-debug
 cmake ..
 make
 ```
 
-A more detailed description is given in the following section.
+A more detailed description is given in the Detailed section below.
+
+## Ubuntu 16.04
+
+If you are using Ubuntu 16.04, you first need to change a single line in the file find_polynomial_roots_jenkins_traub.cc:
+
+```
+cd FieldOpt/ThirdParty/RpolyPlusPlus/src/
+sed -i 's/static constexpr double kRootPairTolerance/static const double kRootPairTolerance/g' find_polynomial_roots_jenkins_traub.cc
```

Comment:
The sed command doesn't seem to be correct. Please replace line 48 with:

sed -i 's/static const double kRelativeTolerance/static constexpr double kRelativeTolerance/g' find_polynomial_roots_jenkins_traub.cc

Reply:
Please disregard my previous comment. I see that after the pull request in the RpolyPlusPlus, this will make FieldOpt code compatible with Ubuntu 16.04